com.unity.multiplayer.mlapi/Runtime/NetworkVariable/INetworkVariable.cs

```diff
@@ -73,5 +73,10 @@ public interface INetworkVariable
         /// </summary>
         /// <param name="behaviour">The behaviour the container behaves to</param>
         void SetNetworkBehaviour(NetworkBehaviour behaviour);
+
+        /// <summary>
+        /// Accessor for the RemoteTick stored in the networkVariable, list, set or dictionary
+        /// </summary>
+        ushort RemoteTick { get; }
```

Comment:
Do you mind describing the problem and the proposed fix in this PR?
I have very little to zero context, what I'm reviewing? :)

Reply:
See below comment